framework/_global.scss

```diff
@@ -26,29 +18,44 @@ body {
   visibility: visible;
 }
 
-.region-sidebar-first {
-  padding: 1.667em 0;
-}
-
 .navbar-toggle {
   border-radius: 0;
-  border: none;
+  border-width: 0;
 }
 
 .navbar-collapse,
 .navbar-fixed-top,
 .navbar {
-  border: none;
+  border-width: 0;
 }
```

Comment:
navbar are related to Bootstrap and we don't use it, so we don't need to write specific rules